spring-cloud-netflix-eureka-client/src/main/java/org/springframework/cloud/netflix/eureka/http/RestTemplateTransportClientFactory.java

```diff
@@ -209,11 +209,6 @@ public HttpStatusCode getStatusCode() throws IOException {
 			return response.getStatusCode();
 		}
 
-		@Override
```

Comment:
We cannot remove public methods in a non-major release. Please revert this change.

Reply:
There is a default implementation on the interface level that delegates to getStatusCode(), so technically nothing is removed.

Reply:
But in any case, I have reverted this change.

Reply:
You're right. Have missed that the implementation is same anyhow. You can remove it.

Reply:
I have re-removed it.